src/Components/Navbar/index.js

```diff
@@ -37,18 +49,37 @@ function Navbar(props) {
       </NavbarBrand>
       <Nav pills>
         <NavItem className={styles.navIcon}>
-          <NavLink to="/notifications">
+          <NavLink href="/notifications">
```

Comment:
I was looking at the documentation for Reactstrap and it shows that it is using href instead of to... 
https://reactstrap.github.io/?path=/docs/components-nav--navs